CHANGELOG.rst

```diff
@@ -6,6 +6,14 @@ All notable changes to the MEF_ELine NApp will be documented in this file.
 [Unreleased]
 ************
 
+[2023.2.9] 2025-01-15
```

Comment:
The latest 2023.2.X release as of Feb 18 is 2023.2.10, this needs to be `2023.2.11`